org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/DeviceRequestTest.java

```diff
@@ -219,19 +237,42 @@ public void testInvalidName() throws Exception {
 	}
 
 	@Test
-	public void testMandelModelConfigure() throws Exception {
-		
+	public void testMandelbrotDeviceInfo() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)